maykin_2fa/templates/maykin_2fa/includes/wizard_form.html

```diff
@@ -1,9 +1,21 @@
 {{ wizard.management_form }}
 
+{% if wizard.form.non_field_errors %}
+     <div class="error-messages">
+        {% for error in wizard.form.non_field_errors %}
+            <p class="errornote">{{ error }}</p>
+        {% endfor %}
+     </div>
+{% endif %}
+
 {% for field in wizard.form %}
-    <div class="form-row {% if field.errors %}form-row--error{% endif %} {% if wizard.steps.current == 'generator' %} centered-text {% endif %}">
-        {{ field.errors }}
+    <div>
```

Comment:
Let's keep this class to apply styling

```suggestion
    <div class="form-row {% if field.errors %}form-row--error{% endif %} {% if wizard.steps.current == 'generator' %} centered-text {% endif %}">
```